apps/examples/testcase/le_tc/kernel/tc_libc_signal.c

```diff
@@ -347,6 +349,75 @@ static void tc_libc_signal_psignal(void)
 	TC_SUCCESS_RESULT();
 }
 
+/**
+* @fn                   :tc_libc_signal_sigwait
+* @brief                :tc_libc_signal_sigwait test sigwait function
+* @scenario             :If signo < 0, it return errno by get_errno().
+*                        Else if signo > 0, return signo that signal number in the location referenced by sig.
+* API's covered         :sigwait()
+* Preconditions         :none
+* Preconditions         :none
+* @return               :void
+*/
+
+static int sigwait_receiver(int argc, FAR char *arvg[])
+{
+	sigset_t set = ALL_SIGNAL_SET;
+	int ret_chk, sig;
+
+	ret_chk = sigwait(&set, &sig);
+
+	if (ret_chk == OK && sig == SIG3) {
+		exit(0);
+	}
+	exit(1);
+}
+
+static int sigwait_sender(int argc, FAR char *arvg[])
+{
+	int ret_chk;
+
+	ret_chk = kill(atoi(arvg[argc-1]), SIG3);
+
+	if (ret_chk == OK) {
+		exit(0);
+	}
+	exit(1);
+}
+
+static void tc_libc_signal_sigwait(void)
+{
+	int recv_pid, snd_pid;
+	FAR char const *in_buf[] = { NULL, NULL };
+	char buf[11];
+	int recv_status, snd_status;
+	int recv_ret, snd_ret;
+	const int PRIORITY = 100;
+	const int STACK_SIZE = 1024;
```

Comment:
1. Is this needed to be local variables instead of definitions?
2. Even if it does, usually CAPITAL is used for definition. Let's use lower characters.

Reply:
modify